types/migrations/20220518174701_edited_collections.sql

```diff
@@ -0,0 +1,28 @@
+-- Add migration script here
+
+-- A file for collections and chapters
+
+create extension if not exists "ltree";
+
+create table collections (
+  id autouuid primary key,
+  collection_name text not null,
+  collection_desc text,
+  slug ltree not null unique -- keep it the same type as chapter ltree
+);
+
+create table chapter (
+  id autouuid primary key,
+  title text not null,
+  document_id uuid references document(id),
+  wordpress_id bigint,
+  index_tree ltree not null unique,
+  slug text not null
+);
+
+create table chapter_attribution (
```

Comment:
If we call the above `collection_chapter`, then let's call this `collection_chapter_attribution`